charts/datadog/templates/container-process-agent.yaml

```diff
@@ -24,13 +24,15 @@
     {{- end }}
     {{- if .Values.datadog.orchestratorExplorer.enabled }}
```

Comment:
This `if` is not required and duplicated with the setting `DD_ORCHESTRATOR_CLUSTER_ID`